MediaBrowser.Api/Sessions/SessionService.cs

```diff
@@ -1,498 +0,0 @@
-using System;
```

Comment:
Why did this get removed?

Reply:
Sorry, it got lost when resolving a merge conflict